### PR TITLE
feat(scripts): support env for scripts

### PR DESCRIPTION
**Pull request by simon-cj**
**What type of PR is this?**
/kind feature
/area serving
/area model

**What this PR does / why we need it**:
Support env for scripts

**Which issue(s) this PR is related to** *(optional, link to 3rd issue(s))*:

Fixes #

Reference to #

**Special notes for your reviewer**:

/cc @FogDong @judgeeeeee 

**Does this PR introduce a user-facing change?**:

```release-note
Support env for scripts
```

### CODE REVIEW

File: scripts/convert/converter/base_convert.py [hunk 1]
```diff
@@ -3,25 +3,28 @@
...
         self.author = data.get('author', None)
 
         # maybe can read inputs in env
-        self.input_value = json.loads(os.getenv(INPUTS_ENV, ''))
-        if 'signature' in data:
+        if self.using_ormbfile:
             self.input_value = data['signature'].get('inputs', [])
             self.output_value = data['signature'].get('outputs', [])
+        else:
+            self.input_value = json.loads(os.getenv(INPUTS_ENV, ''))
```
Comment (judgeeeeee) [inline scripts/convert/converter/base_convert.py:27]:
```suggestion
            self.input_value = json.loads(os.getenv(INPUTS_ENV, '[]'))
```

File: scripts/serving/wrapper/preprocessor.py [hunk 2]
```diff
@@ -37,7 +41,8 @@ def get_critical_env(env):
 
         for env in Preprocessor.env_list:
             setattr(self, '_' + env.lower(), get_critical_env(env))
-
+        
+        self.using_ormbfile = True if os.getenv(USING_ORMBFILE_ENV, '') != "" else False
```
Comment (judgeeeeee) [inline scripts/serving/wrapper/preprocessor.py:45]:
```suggestion
        self.using_ormbfile = True if os.getenv(USING_ORMBFILE_ENV, '') else False
```

File: scripts/serving/wrapper/preprocessor.py [hunk 3]
```diff
@@ -124,11 +129,22 @@ def start(self):
...
+        if self.using_ormbfile:
+            yaml_data = self._extract_yaml()
+            if 'format' in yaml_data.items():
+                logger.error('model format missing')
+                return
+            format = yaml_data["format"]
+        else:
+            inputs = os.getenv(INPUTS_ENV, "")
+            outputs = os.getenv(OUTPUTS_ENV, "")
+            format = os.getenv(FORMAT_ENV, "")
```
Comment (judgeeeeee) [inline scripts/serving/wrapper/preprocessor.py:143]:
```suggestion
            inputs = os.getenv(INPUTS_ENV, '[]')
            outputs = os.getenv(OUTPUTS_ENV, '[]')
            format = os.getenv(FORMAT_ENV, '[]')
```

**Comment by simon-cj** [comment]
ping @FogDong

**Review by FogDong — COMMENTED** [review]
/lgtm

**Pull request merged**